src/test/java/seedu/address/ui/UiPartTest.java

```diff
@@ -84,7 +84,7 @@ private URL getTestFileUrl(String testFilePath) {
 
     /**
      * UiPart used for testing.
-     * It should only be used with invalid FXML files or the valid file located at {@link VALID_FILE_PATH}.
```

Comment:
unintentional delete of whitespace between link and VALID.

This breaks the javadocs generated.